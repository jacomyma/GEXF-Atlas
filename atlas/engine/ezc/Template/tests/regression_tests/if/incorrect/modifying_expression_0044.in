{* file: if/incorrect/modifying_expression_0044.in *}
{var $foo = 42, $bar = 101}
0044:{if ( $foo +  $bar++ )}this should never appear{/if}
