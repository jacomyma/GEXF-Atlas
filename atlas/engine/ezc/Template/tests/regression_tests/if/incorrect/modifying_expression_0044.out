mock:3:26: Expecting an operand without a pre- or post operator.

0044:{if ( $foo +  $bar++ )}this should never appear{/i
                         ^
